gitbutler-ui/src/lib/components/BranchLabel.svelte

```diff
@@ -44,7 +44,8 @@
 		on:blur={() => (inputActive = false)}
 		on:keydown={(e) => {
 			if (e.key == 'Enter') {
-				inputActive = false;
+				// Unmount input field asynchronously to ensure on:change gets executed.
+				setTimeout(() => (inputActive = false), 0);
```

Comment:
Before we merge, can we maybe combine it with the setTimeout on the next line?

Reply:
I tried that, but `label´ is only bound when the `<div/>` is mounted, which only happens after `inputActive` is set to `false`. So unfortunately we have to wait for the next event loop for `label` to not be null.

Reply:
A better solution would probably be to try to avoid unmounting the `input` field at all, but I can see why it was done this way.

Reply:
i suggest we could probably refactor this down the line if needed